recipes/list_items/contact_info/contact_info.stories.js

```diff
@@ -5,7 +5,7 @@ import DtRecipeContactInfo from './contact_info';
 import DtRecipeContactInfoMdx from './contact_info.mdx';
 import DtRecipeContactInfoDefaultTemplate from './contact_info_default.story';
 import DtRecipeContactInfoVariantsTemplate from './contact_info_variants.story';
-import { USER_STATUS_COLOR_MODIFIERS } from './contact_info_constants';
+import { AVATAR_SIZE_MODIFIERS, PRESENCE_STATES_LIST } from '@';
```

Comment:
don't use '@' we had some strange cyclical dependency issues related to this. use `@/components/avatar`